notes-webapp/src/main/webapp/WEB-INF/conf/wiki/ckeditor/config.js

```diff
@@ -55,6 +55,22 @@ CKEDITOR.editorConfig = function (config) {
       items: blocksToolbarGroup
     },
   ];
+  const mobileToolbar = [
```

Comment:
I think that this be done with tis [task](https://community.exoplatform.com/portal/dw/tasks/taskDetail/71945)